api/apis/org_handler.go

```diff
@@ -58,7 +59,7 @@ func (h *OrgHandler) orgCreateHandler(info authorization.Info, r *http.Request)
 	}
 
 	org := payload.ToMessage()
-	org.GUID = uuid.NewString()
+	org.GUID = OrgPrefix + uuid.NewString()
```

Comment:
I notice we are generating the GUID in 3 different layers: handler for orgs and spaces, repo for routes and the app reconciler for processes. Maybe this PR is a good opportunity to move the org and space GUID generation to the repo layer to make things more uniform. The process GUID will have to stay in the app reconciler.

Reply:
Moved the Org/space uuid generation to the repository layer.